rust/agents/scraper/migration/bin/generate_entities.rs

```diff
@@ -0,0 +1,113 @@
+use std::path::Path;
+use std::process::Stdio;
+use std::time::Duration;
+
+use tokio::fs::remove_dir_all;
+use tokio::process::Command;
+use tokio::time::sleep;
+
+use common::*;
+
+mod common;
+
+const RAW_DB_PATH: &str = "./agents/scraper/src/db";
```

Comment:
Wondering if you feel that calling the directory `db` is accurate or if something like `db_entities` is more accurate? I'll defer to you here - for some reason I see the word `db` and I think "DB data probably goes here"

Reply:
I mean, it is in `src` so I would not think it contains the actual database. I could see `entities` but that feels a little more obscure, and `db_entities` feels unnecessarily verbose. I will think on it though as I go forward and maybe something else will come to mind.